src/main/java/vct/importer/JavaLibraryLoader.scala

```diff
@@ -19,4 +19,19 @@ case class JavaLibraryLoader(blameProvider: BlameProvider) extends ExternalJavaL
   } catch {
     case _: FileNotFound => None
   }
+
+  override def loadPkg[G](base: Path, pkg: Seq[String]): Seq[JavaNamespace[G]] = {
+    var resolved = base
+    for(elem <- pkg) {
+      resolved = resolved.resolve(elem)
+    }
+    resolved.toFile.listFiles().map { f =>
```

Comment:
This again seems really dubious, I hope it's not required.

Reply:
Pretty sure it's dead code leftover from old versions